classes/member_show.py

```diff
@@ -0,0 +1,66 @@
+import modules.formattable as format
+
+class MemberShow:
+    def __init__(self, data):
+        self.id = data['id']
+        self.image = format.image_url(self.id)
+        self.first_name = data['first_name']
+        self.last_name = data['last_name']
+        self.role = data['roles'][0]['title']
+        self.phone = data['roles'][0]['phone']
+        self.address = data['roles'][0]['office']
+        self.twitter_url = self.format_twitter_url(data)
+        self.twitter_handle = data['twitter_account']
+        self.youtube = self.youtube_url(data)
+        self.facebook = self.facebook_url(data)
+        self.party = format.party_name(data['current_party'])
+        self.chamber = data['roles'][0]['chamber']
+        self.state = data['roles'][0]['state']
```

Comment:
This is a lot but I guess it has to have this amount of attributes, in a future iteration a many to many for the contact information would be great to look into.

Reply:
We cannot really implement a many to many relationship because it is not a database object.